components/private/Math/src/MDCMath.h

```diff
@@ -30,6 +30,10 @@ static inline CGFloat MDCFabs(CGFloat value) {
 #endif
 }
 
+static inline CGFloat MDCDegreesToRadians(CGFloat degrees) {
+  return degrees * M_PI / 180.f;
```

Comment:
This add an unnecessary cast to float then back to double if CGFloat is double.
This is likely not a big deal, but consider:
 #if CGFLOAT_IS_DOUBLE
  return degrees * M_PI / 180.0;
 #else
  return degrees * M_PI / 180.f;
 #endif

Reply:
Done.